### DISCUSSION TITLE
Switching  3D models

### DISCUSSION THREAD

**Discussion by adrianposteuca** [google/model-viewer#3779] [Q&A]
Hi all,

is there a known issue or is it just my bug? I can not switch 3d models.
Every time that I try the old one is rendered and the new one stays on top.
 
![image](https://user-images.githubusercontent.com/17274973/188457234-d49cfbb7-be69-4328-ae97-27a40a9084ff.png)

![image](https://user-images.githubusercontent.com/17274973/188457558-1ceb3aae-d3d2-4087-953c-6dd4060aaaea.png)

I see there is the same problem in the example provided on Modelviwer.

![image](https://user-images.githubusercontent.com/17274973/188456889-19098bdd-fc46-40dc-b71c-b33b699b02c3.png)

Any workaround?

**Comment by elalish** [comment]
I haven't seen this; can you supply a repro link so we can see what's going on? Also, is this happening on multiple devices/browsers or just one?

**Reply by elalish** [reply]
Nevermind! This is #3780
